flink-python/pyflink/table/table.py

```diff
@@ -817,6 +828,8 @@ def aggregate(self, func: Union[str, Expression]) -> 'AggregatedTable':
             ...                    DataTypes.FIELD("b", DataTypes.INT())]),
             ...               func_type="pandas")
             >>> tab.aggregate(agg(tab.a).alias("a", "b")).select("a, b")
+            >>> # input all columns
+            >>> tab.aggregate(agg.alias("a, b")).select("a, b")
```

Comment:
For Row based operation, the row function should receive Row/DataFrame as input. Otherwise, the fields name information will be missing. This will become a problem when there are hundreds of columns. It may be difficult to use as users have to give each column a meaningful name.